Common/src/main/resources/assets/patchouli/lang/it_it.json

```diff
@@ -1,5 +1,10 @@
 {
 	"item.patchouli.guide_book.name": "Guida",
+	"item.patchouli.guide_book.undefined": "Invalid book: no ID defined",
+	"item.patchouli.guide_book.invalid": "Invalid book: %s",
```

Comment:
not translated?

Reply:
A mistake during comparison, i'll fix it asap